shared/components/ResourceGraph/buildGraph/buildNetworkGraph.js

```diff
@@ -0,0 +1,151 @@
+import { wrap, makeNode, makeEdge, makeRank, makeCluster } from './helpers';
+import { match } from './../relations/relations';
+import { findCommonPrefix } from './../../..';
+
+// layers of network graph
+export const networkFlowKinds = [
```

Comment:
I don't like how layouts like this are defined in the graph code, as it's not extendable. I think configs like that should be saved per-resource (similar to for example how config map templates are stored per resource). Same goes for `networkFlowResources` and `relations`